src/api/generate-csv/map-application-to-csv/helpers/format-date/index.ts

```diff
@@ -0,0 +1,12 @@
+import { format } from 'date-fns';
+
+/**
+ * formatDate
+ * Format a date for design requirement e.g 29 December 2022
+ * @param {Date} timestamp
+ * @param {String} dateFormat defaults to 'd MMMM yyyy' unless provided
+ * @returns {String} Formatted date
+ */
+const formatDate = (timestamp: Date, dateFormat = 'd MMMM yyyy'): string => format(new Date(timestamp), dateFormat);
```

Comment:
Can `dateFormat` please be picked up from a constant?

Reply:
Can we have this as tech improvement ticket outside of this PR please, it's in a few places